adapter/main.go

```diff
@@ -8,22 +8,23 @@ import (
 	"strconv"
 	"time"
 
+	kedav1alpha1 "github.com/kedacore/keda/v2/api/v1alpha1"
+	generatedopenapi "github.com/kedacore/keda/v2/pkg/generated/openapi"
+	prommetrics "github.com/kedacore/keda/v2/pkg/metrics"
+	kedaprovider "github.com/kedacore/keda/v2/pkg/provider"
+	"github.com/kedacore/keda/v2/pkg/scaling"
+	"github.com/kedacore/keda/v2/version"
```

Comment:
Minor nit, could you please move KEDA related imports into a separate block (intended by one line) under the generic imports? As it was before?

Reply:
Sure, fixed them. I have `goimports` running on-file-save, it auto arranged the imports.